backend/users/models.py

```diff
@@ -4,7 +4,11 @@
 User = get_user_model()
 
 
-# Create your models here.
+class LoginGovUser(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
Is a cascade what we want? If we delete a user, that makes sense it should delete the Login.gov info, but is there a case where we want to delete users? Would SACs be unaccessible after this? (Maybe this doesn't matter after they're certified and done through all the phases?)  I ask as maybe this should be `models.PROTECT` instead, but not sure I can answer that.

Reply:
Then this seems fine as it is!